src/coreclr/utilcode/util.cpp

```diff
@@ -513,6 +513,8 @@ BYTE * ClrVirtualAllocWithinRange(const BYTE *pMinAddr,
     return pResult;
 }
 
+#ifdef HOST_WINDOWS
+
 //******************************************************************************
 // NumaNodeInfo
```

Comment:
Delete the `NumaNodeInfo` class?

Reply:
yeah guess there is more cleanup required here, will do as part of a separate PR.

Reply:
I see that you are keeping the VM version of the PAL for Windows under ifdefs. Is there a problem with using the GC PAL on Windows?

Reply:
there shouldnt be an issue with moving windows too, just needs a some more wrangling. Also need to check with @Maoni0 that all numa stuff is handled appropriately in `gcenv.windows`